ivy/functional/ivy/elementwise.py

```diff
@@ -5092,6 +5092,223 @@ def subtract(
     ret
         an array containing the element-wise differences.
 
+         Functional Examples
+    ------------------
+
+    With: code:`ivy.Array` inputs:
+
+    >>> x1 = ivy.array([1, 0, 1, 1])
+    >>> x2 = ivy.array([1, 0, 0, -1])
+    >>> y  =  ivy.subtract(x1, x2)
+    >>> print(y)
+    ivy.array([0, 0, 1, 2])
+
+    >>> x1 = ivy.array([1, 0, 1, 0])
+    >>> x2 = ivy.array([0, 1, 0, 1])
+    >>> y = ivy.subtract(x1, x2)
+    >>> print(y)
+    ivy.array([ 1, -1,  1, -1])
+
+    >>> x1 = ivy.array([1, -1, 1, -1])
+    >>> x2 = ivy.array([0, -1, 1, 0])
+    >>> y = ivy.zeros(4)
+    >>> ivy.subtract(x1, x2, out=y)
+    >>> print(y)
+    ivy.array([ 1,  0,  0, -1])
+
+    >>> x1 = ivy.array([1, -1, 1, -1])
+    >>> x2 = ivy.array([0, -1, 1, 0])
+    >>> y = ivy.subtract(x1, x2, out=x1)
+    >>> print(y)
```

Comment:
It would be more useful if this print statement printed the value of x1, as it would show inplace updates. Also the line after this is misspelled.